tools/update_version.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+if [ $# != 4 ] ; then
+echo "USAGE: $0 major_version minor_version patch_version path"
+echo " e.g.: $0 1 2 3 ."
+exit 1;
+fi
+
+OCCLUM_MAJOR_VERSION=$1
+OCCLUM_MINOR_VERSION=$2
+OCCLUM_PATCH_VERSION=$3
+
+dir=$4
+if [ ! -d $dir ];then
+echo $dir is not dir
+fi
+
+echo "update version to $OCCLUM_MAJOR_VERSION.$OCCLUM_MINOR_VERSION.$OCCLUM_PATCH_VERSION"
+
+cd $dir
+cd src/libos/
+sed -i '/^version =/c\version = "'$OCCLUM_MAJOR_VERSION'.'$OCCLUM_MINOR_VERSION'.'$OCCLUM_PATCH_VERSION'"' Cargo.toml
+sed -i '/^name = "Occlum"/{n;d}' Cargo.lock
+sed -i '/^name = "Occlum"/a\version = "'$OCCLUM_MAJOR_VERSION'.'$OCCLUM_MINOR_VERSION'.'$OCCLUM_PATCH_VERSION'"' Cargo.lock
+cd ../../
+
+cd src/exec/
+sed -i '/^version =/c\version = "'$OCCLUM_MAJOR_VERSION'.'$OCCLUM_MINOR_VERSION'.'$OCCLUM_PATCH_VERSION'"' Cargo.toml
+sed -i '/^name = "occlum_exec"/{n;d}' Cargo.lock
+sed -i '/^name = "occlum_exec"/a\version = "'$OCCLUM_MAJOR_VERSION'.'$OCCLUM_MINOR_VERSION'.'$OCCLUM_PATCH_VERSION'"' Cargo.lock
+cd ../../
+
+cd src/pal/include/
+cat>occlum_version.h<<EOF
```

Comment:
I think there should be a comment to mention that this header is generated automatically by this tool and should not be modified manually.

Reply:
Updated